### ISSUE TITLE
Language of UIAlertViewController Does not change when Reset

### ISSUE THREAD

**Issue by mEldewaik** [MoathOthman/MOLH#50]
When Change Language from english to Arabic Or reverse UIAlertViewController does not localized unless i restart the app please i need fix i faced this issue and tried alot but can not find solution

**Comment by abdallahnh** [comment]
@mEldewaik did you tried UIAlertController instead, I'm using it and it is fine.

**Comment by mEldewaik** [comment]
@abdallahnh i use UIAlertViewController to share text does UIAlertController do the same ?

**Comment by abdallahnh** [comment]
@mEldewaik yes sure it does

**Comment by mEldewaik** [comment]
@abdallahnh  the code please 🙏

**Comment by MoathOthman** [comment]
@mEldewaik can you share a screenshot of what are you trying to localize.  I guess you mean UIActivityViewController not UIAlertViewController as there is no such think AFAIK!

**Comment by mEldewaik** [comment]
@MoathOthman yes i mean UIActivityViewController it can not be localized unless i close the app and open it again

**Comment by MoathOthman** [comment]
I don\t there is a straight way  to achieve it since its part of uikit and ,  check https://medium.com/@dark_torch/working-with-localization-in-swift-part-2-e7c8a660eb2a on "Other Frameworks and bundles" section , at best you get to change the localization but not the layout.

**Comment by mEldewaik** [comment]
@MoathOthman  i don't think it is the answer as it is already translated but its is not flipped and translated until i close and open the app again .

**Comment by MoathOthman** [comment]
@mEldewaik I don't think it is possible, but I will look into it soon.

**Comment by mEldewaik** [comment]
@MoathOthman  so can i disable localization for UIActivityViewController and use only english in all cases ?

**Comment by MoathOthman** [comment]
no because you don't have access to the layout code and the localization is handled in another bundle. 

if you can find the localization keys for the strings in the activityview , then there is a way. but i forgot tbh how to do that, if you managed to do that please let me know.

**Comment by mEldewaik** [comment]
@MoathOthman i saw before application support english and arabic and the UIActivityViewController only in english

**Comment by MoathOthman** [comment]
it would be interesting to be able to force certain language for certain strings,  or even certain view controllers. am not aware of any currently, but will look into it.

**Comment by abdallahnh** [comment]
@mEldewaik i added some appearance changes please try from master and let me know if this fixed your issue

**Comment by mEldewaik** [comment]
@abdallahnh so how can i test it please can you advice ?
how can i try your fix ?

**Comment by mEldewaik** [comment]
@abdallahnh i download the pod and implement button and make a share but the issue still exist i can not find solution :(

**Comment by Emy87aa** [comment]
@mEldewaik I think you may when building your app at the start, you build it as (Arabic) in Storyboard when the app default language is (en) in xcode , so that it is verse the language wrong. 
you should set the language of app to be (ar) first from (Edit Scheme) when build Arabic storyboard
<img width="1051" alt="Screen Shot 2020-02-27 at 12 26 04 PM" src="https://user-images.githubusercontent.com/1811875/75435968-7996a300-595c-11ea-81ce-1163be3de858.png">
